tests/integration/suite/test_users.py

```diff
@@ -16,3 +18,62 @@ def test_user_cant_deactivate_self(admin_mc):
         client.update(admin_mc.user, enabled=False)
 
     assert e.value.error.status == 422
+
+
+def test_globalrolebinding_create(admin_mc, remove_resource):
```

Comment:
Nit: update the name to reflect this is part of checking cleanup. A comment describing that would be good as well. `test_globalrolebinding_finalizer_cleanup` or similar.